_overviews/reflection/overview.md

```diff
@@ -132,7 +132,7 @@ The second step involves obtaining a `ClassMirror` for class `Person` using
 the `reflectClass` method. The `ClassMirror` provides access to the
 constructor of class `Person`.
 
-    scala> val ctor = ru.typeOf[Person].declaration(ru.nme.CONSTRUCTOR).asMethod
+    scala> val ctor = ru.typeOf[Person].decls(ru.nme.CONSTRUCTOR).asMethod
```

Comment:
@SethTisue This PR introduced a bug, replacing `declaration` (singular) with `decls` (plural) here and on line 174 below. A [fix](https://github.com/scala/docs.scala-lang/pull/935/files#diff-de3b0e68fa6f7e4e210778b35252b9e2L134) is part of #935